packages/server/safeMutations/acceptTeamInvitation.ts

```diff
@@ -61,8 +57,11 @@ const acceptTeamInvitation = async (teamId: string, userId: string) => {
           .getAll(userId, {index: 'userId'})
           .filter({removedAt: null})
           .coerceTo('array')
-      }) as unknown) as User & {organizationUsers: OrganizationUser[]}
-  }).run()
+      })
+      .run(),
+    getTeamsById([teamId])
+  ])
+  const team = teams[0] ?? null
```

Comment:
here you're saying team could be null, but in the next line you're accessing team. that'll cause a nasty runtime error if it really is null. (there are a handful of other places this happens)